exasol/secret_store.py

```diff
@@ -151,3 +151,11 @@ def items(self) -> Iterable[Tuple[str, str]]:
             res = cur.execute(f"SELECT key, value FROM {TABLE_NAME}")
             for row in res:
                 yield row[0], row[1]
+
+    def remove(self, key: str) -> None:
+        """
+        Deletes entry with the specified key if it exists.
+        Doesn't raise any exception if the key doesn't exist.
+        """
+        with self._cursor() as cur:
+            cur.execute(f"DELETE FROM {TABLE_NAME} WHERE key=?", [key])
```

Comment:
We probably should quote TABLE_NAME properly to be on the safe side, but this seems a issue with all queries. So maybe create ticket for it.

Reply:
Yes, we need to make this change throughout the module. I will create a ticket for that.